lerobot/common/datasets/_video_benchmark/run_video_benchmark.py

```diff
@@ -13,6 +13,21 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
+"""Assess the performance of video decoding in various configurations.
+
+This script will run different video decoding benchmarks where one parameter varies at a time.
+These parameters and theirs values are specified in the BENCHMARKS dict.
+
+All of these benchmarks are evaluated within different timestamps modes corresponding to different frame-loading scenarios:
+    - `1_frame`: 1 single frame is loaded.
+    - `2_frames`: 2 consecutive frames are loaded.
+    - `2_frames_4_space`: 2 frames separated by 4 frames are loaded.
```

Comment:
Looking at the code, I think they are actually separated by 3 frames

Reply:
@aliberts could you confirm?

Reply:
Indeed, nice catch!
Fixed https://github.com/huggingface/lerobot/pull/220/commits/bf3dbbdca6170779f49bfe1a72011ddb6e6af570